test/extended/util/image/README.md

```diff
@@ -36,6 +36,8 @@ To become an OWNER in this directory you must be given permission to push to thi
 
 * Temporary
   * `docker.io/summerwind/h2spec:2.4.0` - for HTTP2 testing, waiting for mirror
+  * `docker.io/openshifttest/test-build-roots2i:latest` - build e2e's need to verify that a s2i builder image which
```

Comment:
I'm thinking this will need to change to `quay.io/openshift/community-e2e-images:test-build-roots2i` after re-reading https://github.com/openshift/origin/tree/master/test/extended/util/image#to-add-a-new-image but after 

1) https://github.com/openshift/release/pull/14178 merges 
2) and some of @bparees 's asks for clarification from @smarterclayton in https://github.com/openshift/origin/pull/24887#discussion_r538841935 and https://github.com/openshift/origin/pull/24887#discussion_r538842654 get sorted out

Reply:
ok https://github.com/openshift/release/pull/14178 has merged